spec/RDS.rds.shouldMatchPropertyValue.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { RDS } from '../src/RDS'
+import { callbackSuccessReturning, nvp } from './support'
+import { testRds } from './RDS.stub'
+var AWSMock = require('aws-sdk-mock')
+var sinon = require('sinon')
+
+describe("RDSObject#shouldMatchPropertyValue function", () => {
```

Comment:
The focus should be on behaviour, over checking configuration.

These may still be useful, but less so than behavioural tests.

Reply:
Agree.

But from an implementation PoV, you could check the configuration as a unit test - e.g. check cloudformation output.

Depends on feedback.

From a PR PoV - this isn't a blocker and no point removing it.